recipe/meta.yaml

```
@@ -47,7 +47,8 @@ about:
   license: MIT
   license_family: MIT
   license_file: LICENSE
-  summary: Python port of markdown-it. Markdown parsing, done right!
+  description: |
```

```suggestion
  summary: Python port of markdown-it. Markdown parsing, done right!
  description: |
```

I would still leave the summary, even if it's the same content.

---

Got it thanks!